server/src/main/java/org/apache/druid/server/coordinator/loading/SegmentReplicaCountMap.java

```diff
@@ -49,6 +49,10 @@ private void initReplicaCounts(DruidCluster cluster)
     cluster.getHistoricals().forEach(
         (tier, historicals) -> historicals.forEach(
             serverHolder -> {
+              if (serverHolder.isUnmanaged()) {
+                // Don't manage
```

Comment:
```suggestion
                // Don't count replicas since this is an unmanaged server
```

Reply:
Already changed this comment to `// Don't count segments on unmanaged historicals towards replica counts.`. Is this fine?

Reply:
Yes, that works too. thanks!